Misc/NEWS.d/next/Library/2019-09-16-21-47-48.bpo-38155.d92lRc.rst

```diff
@@ -0,0 +1 @@
+Add __all__ to the datetime module, including unit tests.
```

Comment:
Made a couple of minor formatting adjustments and removed the part about the tests. News entries typically don't contain information about the tests, as they aren't part of the public API. 
```suggestion
Add ``__all__`` to  :mod:`datetime`.
```